src/platform-aws.cpp

```diff
@@ -479,25 +499,15 @@ int platform_init(const char **provider_filter)
 		(FI_MAJOR(libversion) > 1 || (FI_MAJOR(libversion) == 1 && FI_MINOR(libversion) >= 13))
 		? "FI_EFA_FORK_SAFE"
 		: "RDMAV_FORK_SAFE";
-	if (!getenv(fork_safe_var_name)) {
-		NCCL_OFI_INFO(NCCL_INIT, "Setting %s environment variable to 1", fork_safe_var_name);
-		ret = setenv(fork_safe_var_name, "1", 1);
-		if (ret != 0) {
-			NCCL_OFI_WARN("Unable to set %s", fork_safe_var_name);
-			ret = -errno;
-			goto exit;
-		}
-	}
+	env_manager::getInstance().insert_envvar(fork_safe_var_name, "1", false);
```

Comment:
We removed the `NCCL_OFI_INFO` logs for setting the `fork_safe_var_name`/ `NCCL_NET_FORCE_FLUSH`/`NCCL_NVLSTREE_MAX_CHUNKSIZE`/`NCCL_NVLS_CHUNKSIZE`, which keeping the logs for setting `NCCL_NVLS_ENABLE`  and `NCCL_TOPO_FILE`. Would we want to keep the `NCCL_OFI_INFO` logs for setting all the envs, or other select ones?

Reply:
We didn't remove them, we just delayed them until they're actually jammed into the environment during `update_environment()`.  But some of them have additional data than the environment change in the print, so I left those.  Not perfect, but seemed like a reasonable compromise.